tests/functional/test_basic.py

```diff
@@ -21,7 +21,7 @@
 from PyInstaller.utils.tests import importorskip, skipif_win, skipif_winorosx, skipif_notwin
 
 # Directory with data for some tests.
-_DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
+from tests.functional.data_dir import DATA_DIR
```

Comment:
Done. I also placed it on pyi_get_datadir.py (one location for testing, another for code under test).